client/branded/src/search-ui/results/StreamingSearchResultsList.tsx

```diff
@@ -267,7 +267,7 @@ export const StreamingSearchResultsList: React.FunctionComponent<
             <VirtualList<SearchMatch>
                 as="ol"
                 aria-label="Search results"
-                className={classNames('mt-2 mb-0', styles.list)}
+                className={classNames(styles.list)}
```

Comment:
```suggestion
                className={styles.list}
```

We can safely remove the `classNames` function call since we aren't merging styles anymore.

Reply:
Thanks!